src/components/ItaliaTheme/Blocks/RssBlock/CardWithoutImageRssTemplate.jsx

```diff
@@ -56,11 +56,9 @@ const CardWithoutImageRssTemplate = ({
                 <Card noWrapper={false} tag="div" className="card-bg">
                   <CardBody tag="div">
                     <div className="category-top">
-                      {item?.categories?.length > 0 && item.categories[0]._ && (
+                      {item?.source?.length > 0 && (
```

Comment:
item.categories deve rimanere.
In più, deve essere mostrata anche la sorgente.
Mostrerei prima la sorgente e poi le categorie.